console/src/test/java/nl/nn/adapterframework/management/web/ApiExceptionTest.java

```diff
@@ -28,8 +28,8 @@ public class ApiExceptionTest {
 
 	public static List<?> data() {
 		return Arrays.asList(new Object[][] {
-			{"cannot configure", new ConfigurationException("cannot configure")},
```

Comment:
Waarom mag dit geen `ConfigurationException` meer zijn?

Zullen we `ConfigurationException` ook meteen meenemen naar iaf-commons? (Er zitten verder geen extra afhankelijkheden daarin en ik vind het wel een logische exceptie om te kunnen gooien).

Reply:
Ik vind dat een ConfigurationException / ListenerException / SenderException ed. meer thuishoren in core. Tot we die classes ergens anders nodig hebben denk ik dat we ze beter daar kunnen laten staan.

Reply:
ListenerException etc zeker, ik dacht dat ConfigurationException misschien nog algemeen genoeg was. Maar als Console dat nog niet gebruikt maar alleen de test het gebruikt, dan prima, hoeft nog niet over.